android/KMEA/app/src/main/java/com/keyman/engine/util/WebViewUtils.java

```diff
@@ -51,6 +61,33 @@ public static EngineWebViewVersionStatus getEngineWebViewVersionStatus(
     return EngineWebViewVersionStatus.FULL;
   }
 
+  /**
+   * Determine the state of the system WebView (required for Keyman)
+   * @param context
+   * @return {SystemWebViewStatus}
+   */
+  public static SystemWebViewStatus getSystemWebViewStatus(
+      Context context) {
+    // Check whether WebView is installed - this also returns true when installed but disabled
```

Comment:
returns true?

Reply:
Will clarify the comment
```suggestion
    // Check whether WebView is installed
    // If the package is installed but disabled, hasSystemFeature() still returns true
```